windows/src/developer/TIKE/project/Keyman.Developer.System.Project.ProjectLoader.pas

```diff
@@ -177,7 +177,13 @@ procedure TProjectLoader.LoadUser;   // I4698
 begin
   if not FileExists(ChangeFileExt(FFileName, Ext_ProjectSourceUser)) then Exit;
 
-  doc := LoadXMLDocument(ChangeFileExt(FFileName, Ext_ProjectSourceUser)); //TXMLDocument.Create(nil);
+  try
+    doc := LoadXMLDocument(ChangeFileExt(FFileName, Ext_ProjectSourceUser));
+  except
+    on E:Exception do
+      raise EProjectLoader.Create('Error loading project user settings file: '+E.Message);
```

Comment:
What does the user see after `raise`?

Reply:
This is caught at the UI level when the file is loaded and shown in a message box.

Reply:
In the command line compiler it would be shown as an console error message and would abort the build.